frontend/src/components/Pages/InspectionPage/InspectionOverview.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, Tabs, Icon } from '@equinor/eds-core-react'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { RefreshProps } from '../FrontPage/FrontPage'
+import { DeckMissionType, InspectionSection, OngoingMissionType } from './InspectionSection'
+import { useEffect, useRef, useState } from 'react'
+import { BackendAPICaller } from 'api/ApiCaller'
+import { AllInspectionsTable } from './InspectionTable'
+import { getInspectionDeadline } from 'utils/StringFormatting'
+import { Inspection } from './InspectionSection'
+import styled from 'styled-components'
+import { useContext } from 'react'
+import { InstallationContext } from 'components/Contexts/InstallationContext'
+import { Icons } from 'utils/icons'
+import { MissionStatus } from 'models/Mission'
+
+const StyledButton = styled(Button)`
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    border-radius: 4px;
+`
+
+const StyledContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: end;
+    gap: 20px;
+`
+
+const StyledView = styled.div`
+    display: flex;
+    align-items: flex-start;
+`
+
+export function InspectionOverviewSection({ refreshInterval }: RefreshProps) {
+    const { TranslateText } = useLanguageContext()
+    const [activeTab, setActiveTab] = useState(0)
+    const [allMissions, setAllMissions] = useState<Inspection[]>()
+    const installationCode = useContext(InstallationContext).installationCode
+    const echoURL = 'https://echo.equinor.com/missionplanner?instCode='
+    const anchorRef = useRef<HTMLButtonElement>(null)
+    const [ongoingMissions, setOngoingMissions] = useState<OngoingMissionType>({})
+    const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
```

Comment:
```suggestion
    const [activeTab, setActiveTab] = useState(0)
    const [allMissions, setAllMissions] = useState<Inspection[]>()
    const [ongoingMissions, setOngoingMissions] = useState<OngoingMissionType>({})
    const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
    const anchorRef = useRef<HTMLButtonElement>(null)
    const installationCode = useContext(InstallationContext).installationCode
    const echoURL = 'https://echo.equinor.com/missionplanner?instCode='
```

Move useStates together for better readability